### PR TITLE
add content to open and closed tabs on the dashboard

### PR DESCRIPTION
**Pull request by MaryanaMysak**
Знімок екрана 2021-08-11 о 17 57 52

### CODE REVIEW

File: src/components/challenge/component.js [hunk 1]
```diff
@@ -3,35 +3,42 @@ import styles from "./_.module.css";
...
                 <img src={userImg} alt="Winner" className={styles.winnerImg} />
-                <span className={styles.winnerName}>Loulou won the challenge</span>
+                <span className={styles.winnerName}>{winnerName} won the challenge</span>
               </>
             )}
             <span className={styles.votes}>0 votes</span>
-            {isOpen && (
+            {!winnerName && (
               <button className={styles.button}>
                 <img src={voteIcon} alt="Vote" />
```
Comment (StepanNaryshkovEleks) [inline src/components/challenge/component.js:43]: should we check a role of the user?

File: src/pages/dashboard/component.js [hunk 2]
```diff
@@ -42,17 +42,26 @@ export const Dashboard = ({fetchChallenges, challenges, user}) => {
...
-              challenges.map((challenge, i) => (
-                <Challenge
-                  key={i}
-                  data={{...challenge, username: user.actorHandle.assetId.id}}
-                  challengeIndex={i}
-                />
-              ))}
+            {challenges.active.map((challenge, i) => (
+              <Challenge
+                key={i}
```
Comment (StepanNaryshkovEleks) [inline src/pages/dashboard/component.js:47]: please, use id of the challenge
Example is here https://github.com/StepanNaryshkovEleks/pwa/blob/main/src/pages/challenges/component.js#L76

File: src/redux/sagas/challenge.js [hunk 3]
```diff
@@ -142,82 +142,82 @@ export function* getChallenges(props) {
...
+        } else if (
+          indx >= 0 &&
+          detailedChallengesResponse[i].challengePotential.challengeState
+            .selectParticipantEntryArray.length !== 0
+        ) {
+          newData.closed.push(detailedChallengesResponse[i]);
+        } else if (
+          indx >= 0 &&
+          detailedChallengesResponse[i].challengePotential.challengeState
+            .participantArray[indx].participantStatus === "ENGAGED"
```
Comment (StepanNaryshkovEleks) [inline src/redux/sagas/challenge.js:206]: should we check here if there is any winner?

**Comment by MaryanaMysak** [comment]
Fixed comments

**Pull request merged**